src/Recruiter/Worker/Repository.php

```diff
@@ -45,24 +47,26 @@ public function deadWorkers($consideredDeadAt)
     {
         return $this->roster->find(
             ['last_seen_at' => [
-                '$lt' => new MongoDate($consideredDeadAt->getTimestamp())]
+                '$lt' => new MongoUTCDateTime($consideredDeadAt->format('U') * 1000)]
```

Comment:
It's not well documented but MongoUTCDateTime can directly take a `DateTime` or `DateTimeImmutable` in its constructor

Reply:
but the result is different:
```
php > $datetime = new DateTime();
php > $x = new MongoDB\BSON\UTCDateTime($datetime);
php > $y = new MongoDB\BSON\UTCDateTime($datetime->format('U') * 1000);
php > 
php > var_export($d);
DateTime::__set_state(array(
   'date' => '2019-04-18 10:49:21.385441',
   'timezone_type' => 3,
   'timezone' => 'Europe/Berlin',
))
php > 
php > var_export($x);
MongoDB\BSON\UTCDateTime::__set_state(array(
   'milliseconds' => '1555577769571',
))
php > 
php > var_export($y);
MongoDB\BSON\UTCDateTime::__set_state(array(
   'milliseconds' => '1555577769000',
))
```